toolchain/base/value_store.h

```diff
@@ -177,6 +177,9 @@ class ValueStore
 
   ValueStore() = default;
   explicit ValueStore(IdTag tag) : tag_(tag) {}
```

Comment:
Similarly, this one? (but this one still being in use seems a little more likely than CanonicalValueStore, since ValueStore is used in more spots)